gwsumm/config.py

```diff
@@ -49,7 +48,7 @@
     'GWSummConfigParser',
 ]
 
-CONFIGDIR = os.path.join(sys.prefix, 'etc', 'gwsumm', 'configuration')
+CONFIGDIR = os.path.join(os.path.dirname(__file__), '..', 'share')
```

Comment:
This, I think, only works when running directly from the `git` project, which isn't the normal practice.

Reply:
Oh, hm, I would have thought this should fix a problem that `CONFIGDIR` is pointing to an invalid location for anyone not running in production. Maybe it's best I revert this line and try it out in a different PR

Reply:
If you _don't_ update this line, it _will_ be broken for people running in production (because the configuration files are no-longer distributed into the referenced directory).

If you _do_ update this line as is, it _will_ still be broken for people running in production because `../share` will not exist.

Reply:
Gotcha, thanks. Also looking further at this gives me some other things to think about for improvements, but let's wait on that for another PR.

Reply:
Fixed as suggested in the packaging of ini files comment